tests/test_parquet.py

```diff
@@ -0,0 +1,36 @@
+import io
+
+import astropy.table as astrotab
+import numpy as np
+import pyarrow.parquet as pq
+import pytest
+
+from roman_datamodels import datamodels
+from roman_datamodels import maker_utils as utils
+
+source_catalogs = [
+    (datamodels.ImageSourceCatalogModel, utils.mk_image_source_catalog),
+    (datamodels.MosaicSourceCatalogModel, utils.mk_mosaic_source_catalog),
+]
+
+
+@pytest.mark.parametrize(("catalog_class", "mk_catalog"), source_catalogs)
+def test_source_catalog(catalog_class, mk_catalog):
+    sc_node = mk_catalog(save=False)
+    sc_dm = catalog_class(sc_node)
+    bio = io.BytesIO()
+    sc_dm.to_parquet(file=bio)
+    bio.seek(0)
+
+    ptab = astrotab.Table.read(bio, format="parquet")
```

Comment:
```suggestion
    ptab = pq.ParquetFile(bio).read().to_pydict()
```
Does this work to allow this PR to avoid adding pandas as a dependency?